translations/binaries/slps_strings_group_08.json

```
@@ -180,28 +180,28 @@
             "string": "リーダーの背後の敵を優先して[LINE]\n狙います。[END]",
             "original_bytes_length": 40,
             "available_bytes_size": 40,
-            "translation": "리더의 뒤쪽 적을 우선하여[LINE]\n노립니다.[END]"
+            "translation": "리더 배후의 적을 우선하여[LINE]\n노립니다.[END]"
         },
         {
             "offset": 1708768,
             "string": "背中は預けた[END]",
             "original_bytes_length": 13,
             "available_bytes_size": 15,
-            "translation": "등은 맡겼다[END]"
+            "translation": "등 뒤를 부탁한다[END]"
         },
         {
             "offset": 1708784,
             "string": "近接攻撃と晶術・間接攻撃をパーティ[LINE]\n構成や戦況に応じて切り替えます。[END]",
             "original_bytes_length": 68,
             "available_bytes_size": 71,
-            "translation": "근접 공격과 결정술・간접 공격을 파티[LINE]\n구성이나 전황에 맞춰 전환합니다.[END]"
+            "translation": "근접 공격과 정술・간접 공격을 파티[LINE]\n구성이나 상황에 맞춰 전환합니다.[END]"
         },
         {
             "offset": 1708856,
             "string": "攻守の要となれ[END]",
             "original_bytes_length": 15,
             "available_bytes_size": 15,
-            "translation": "공수의 요체가 되어라[END]"
+            "translation": "공수의 요소가 되어라[END]"
```

```suggestion
            "translation": "공수의 핵심이 되어라[END]"
```

'요소'보다 '핵심'이 더 많이 사용되는 한국어 표현일 것 같습니다~